webpack.config.js

```diff
@@ -6,10 +6,14 @@ module.exports = {
     module: {
         rules: [
             {
-                test: /\.tsx?$/,
+                test: /\.js?$/,
```

Comment:
This is telling webpack to use ts-loader for js files, but not ts files. Can you change it so it targets both?